htdocs/compta/paiement.php

```diff
@@ -369,7 +369,7 @@ function setPaiementCode()
 			                    }
             					if ($(\'#fieldchqemetteur\').val() == \'\')
             					{
-            						var emetteur = ('.$facture->type.' == 2) ? \''.dol_escape_htmltag($conf->global->MAIN_INFO_SOCIETE_NOM).'\' : jQuery(\'#thirdpartylabel\').val();
+            						var emetteur = ('.$facture->type.' == 2) ? \''.dol_escape_js(dol_escape_htmltag($conf->global->MAIN_INFO_SOCIETE_NOM)).'\' : jQuery(\'#thirdpartylabel\').val();
```

Comment:
Why keeping the dol_escape_htmltag.
We are clearly in js, so escaping must be dol_escape_js. No need for dol_escape_htmltag.